src/commands/init.ts

```diff
@@ -10,6 +11,10 @@ export default class Init extends FlanCommand {
 
   static flags = {
     ...FlanCommand.flags,
+    confirm: flags.boolean({
+      char: "y",
```

Comment:
should be `--yes` with `-y` as alias, similar to `apt`: https://askubuntu.com/questions/672892/what-does-y-mean-in-apt-get-y-install-command